packages/create/template/docs/zh/demo/markdown.md

```diff
@@ -254,6 +254,29 @@ $$
 
 - [查看详情](https://theme-hope.vuejs.press/zh/guide/markdown/vue-playground.html)
 
+#### Sandpack 交互演示
+
+::: sandpack#vue Vue 示例
+
+@file /src/App.vue
+
+```vue
+<script setup>
+import { ref } from "vue";
+
+const msg = ref("Hello Playground!");
+</script>
+
+<template>
+  <h1>{{ msg }}</h1>
+  <input v-model="msg" />
+</template>
+```
+
+:::
+
+- [查看详情](https://theme-hope.vuejs.press/zh/guide/markdown/sandpack.html)
+
```

Comment:
Heavy features like this should not be added in template, instead a iframe should be used